flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/catalog/hive/HiveCatalogHiveMetadataTest.java

```diff
@@ -70,6 +77,35 @@ public void testCreateTable_StorageFormatSet() throws Exception {
 		assertFalse(StringUtils.isNullOrWhitespaceOnly(serde));
 	}
 
+	// ------ table and column stats ------
+	@Test
+	public void testAlter_GET_TableColumnStatistics() throws Exception {
+		catalog.createDatabase(db1, createDb(), false);
+		CatalogTable catalogTable = createTable();
+		catalog.createTable(path1, catalogTable, false);
+		Map<String, CatalogColumnStatisticsDataBase> columnStatisticsDataBaseMap = new HashMap<>();
+		columnStatisticsDataBaseMap.put("first", new CatalogColumnStatisticsDataString(10, 5.2, 3, 100));
+		columnStatisticsDataBaseMap.put("second", new CatalogColumnStatisticsDataLong(0, 1000, 3, 0));
+		columnStatisticsDataBaseMap.put("third", new CatalogColumnStatisticsDataString(15, 5.2, 3, 50));
+		CatalogColumnStatistics catalogColumnStatistics = new CatalogColumnStatistics(columnStatisticsDataBaseMap);
+		catalog.alterTableColumnStatistics(path1, catalogColumnStatistics, false);
+		CatalogTestUtil.checkEquals(catalogColumnStatistics, catalog.getTableColumnStatistics(path1));
```

Comment:
nit: add an empty line to separate logic code and assertions to improve readability?